src/display/metadata.js

```diff
@@ -89,22 +89,24 @@ class Metadata {
     return entry.childNodes.filter(node => node.nodeName === "rdf:li");
   }
 
-  _getCreators(entry) {
-    if (entry.nodeName !== "dc:creator") {
-      return false;
-    }
+  _getArray(entry) {
+    const name = entry.nodeName;
+    assert(
```

Comment:
This `assert` is useless since the name has already been checked in the `switch`.

Reply:
I added that as an extra level of security, to avoid a future accidental call of `_getArray` for an unsupported name.
However, as you pointed it out, it's not entirely necessary and I've removed it now.

Reply:
Since the return value is now consistent, I've renamed/rewritted the method to `_parseArray` instead.